src/Microsoft.AspNetCore.Sockets/TransportType.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Microsoft.AspNetCore.Sockets
+{
+    [Flags]
+    public enum TransportType
+    {
+        /// <summary>
+        /// Every transport
+        /// </summary>
+        All = Streaming | LongPolling,
```

Comment:
As per our conversation a few days ago - Transport should be common so that we don't have to copy this enum on the client. Also, All/Streaming should not be part of this enum. Finally I understand why this enum is flags but it's a bit weird.